inc/functions_search.php

```diff
@@ -1592,7 +1593,7 @@ function perform_search_mysql_ft($search)
 
 	$thread_prefixcut = '';
 	$prefixlist = array();
-	if($search['threadprefix'] && $search['threadprefix'][0] != 'any')
+	if(!empty($search['threadprefix']) && $search['threadprefix'][0] != 'any')
```

Comment:
It might be sensible to check that $search['threadprefix'] is an array unless this is already done somewhere else.

Reply:
This is fixed, thanks!

Reply:
Fixed. Thanks!